### PR TITLE
feat(payment): STRIPE-82 Add code to show images on the list

### PR DESCRIPTION
**Pull request by EliLV21**
## What? [STRIPE-82](https://bigcommercecloud.atlassian.net/browse/STRIPE-82)

Add images between list for payments providers

## Why?

Display the payment method’s logo next the payment method radio selector label for the following payment methods

## Testing / Proof

Screen Shot 2022-07-22 at 15 49 17

Screen Shot 2022-07-22 at 16 06 32

@bigcommerce/checkout @bigcommerce/apex-integrations

### CODE REVIEW

File: packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ function getPaymentMethodTitle(
     const cdnPath = (path: string) => `${basePath}${path}`;
 
     return method => {
+        const paymentWithLogo = method.initializationData.methodsWithLogo !== undefined ? method.initializationData.methodsWithLogo : [];
```
Comment (AlanMunozN) [inline packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx:33]:
```suggestion
        const paymentWithLogo = method.initializationData?.methodsWithLogo ? method.initializationData.methodsWithLogo : [];
```
OR

```suggestion
        const paymentWithLogo = method.initializationData && method.initializationData.methodsWithLogo ? method.initializationData.methodsWithLogo : [];
```

Comment (EliLV21) [reply packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx]: Hi Alan, this change is already do it, can you validate again the code, please?

**Review by AlanMunozN — APPROVED** [review]
LGTM_prim

**Review by animesh1987 — APPROVED** [review]

**Review by bc-peng — APPROVED** [review]

**Comment by BCHMercado** [comment]
💚  testing done
https://bigcommercecloud.atlassian.net/jira/software/c/projects/STRIPE/boards/622?modal=detail&selectedIssue=STRIPE-82

**Pull request merged**